cli/scripts/mecha-comet-gen1-r5.auto

```diff
@@ -0,0 +1,47 @@
+SDP: boot -f _flash.bin -scanlimited 0x800000
```

Comment:
we will have to heavily document this script, add explanation for every line possible